hw/ip/rv_dm/data/rv_dm_debug_mem.hjson

```diff
@@ -0,0 +1,162 @@
+// Copyright lowRISC contributors.
```

Comment:
@tjaychen @msfschaffner @GregAC 
Would one of you please review this HJson file I wrote to reflect the debug memory within rv_dm? This helps us generate a proper RAL model that we cause effectively within our DV framework.